GeoConnector.py

```diff
@@ -1,24 +1,27 @@
 import googlemaps
+import sys
 from datetime import datetime
 
 class GeoConnector:
 	OK_STATUS = 'OK'
 	WALKING_MODE = 'walking'
+	DRIVING_MODE = 'driving'
 	MILES_TO_KM_RATIO = 0.6213
 
 	def __init__(self, service_key):
 		self.client = googlemaps.Client(service_key)
 
-	def get_driving_time(self, start_pt, end_pt):
-		return 0
-
-	def get_walking_dist(self, start_pt, end_pt):
-		'''Get the walking distance from start_pt to end_pt.
-		If walking between the two points is not possible, None is returned.
+	def get_dist(self, start_pt, end_pt, mode):
+		'''Get the distance from start_pt to end_pt. Mode must either be WALKING_MODE
+		or DRIVING_MODE. If traveling between the two points is not possible, None is returned.
 		Returns a 2 element tuple where the first element is the travel duration in
 		minutes and the 2nd is the distance in miles.
 		'''
-		distance_matrix = self.client.distance_matrix(start_pt, end_pt, self.WALKING_MODE)
+		if mode != self.WALKING_MODE and mode != self.DRIVING_MODE:
+			print >> sys.stderr, 'Invalid mode: ' + mode
```

Comment:
I"m happy just to use print. Presumably, when we move past the mvp we will setup actual logging.